README.md

```diff
@@ -3,6 +3,40 @@ React Native Tipsi custom UI elements
 
 ## Components
 
+### `<Counter />`
+
+Component to change the number by press "+" or "-".
+
+#### Counter Props
+
+| Name | Desc | Type | Default
+| --- | --- | --- | --- |
+| `step` | Step of counting.  | Number | `1`
+| `startValue` | Initial value of counter. | Number | `1`
+| `onValueChange` | Handle value changes. | Function | `undefined`
```

Comment:
default value for `onValueChange` should be empty function `() => {}`, not *undefined*